redislock.go

```diff
@@ -70,12 +93,15 @@ func (c *Client) Obtain(ctx context.Context, key string, ttl time.Duration, opt
 	}
 
 	var ticker *time.Ticker
+	for i := range keys {
+		keys[i] = "redislock_" + keys[i]
```

Comment:
Auto-generating prefix - I don't like that.

Let user decide. Just use what user provides.

Doesn't prevent us from merging and then polishing it ourselves before tagging a release.

Reply:
agree, we should not impose any limitations